src/routes/index.jsx

```diff
@@ -1,11 +1,16 @@
 import React, { Component } from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+import * as routes from '../constants/routes';
```

Comment:
I think relative paths are a good idea only when the files are collocated and unlikely to be split.